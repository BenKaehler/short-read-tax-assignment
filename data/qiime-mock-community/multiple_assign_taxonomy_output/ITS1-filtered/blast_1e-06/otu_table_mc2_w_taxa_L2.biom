{"id": "None","format": "Biological Observation Matrix 1.0.0","format_url": "http://biom-format.org","type": "Taxon table","generated_by": "QIIME 1.6.0-dev","date": "2013-06-14T08:25:36.623950","matrix_type": "sparse","matrix_element_type": "float","shape": [4, 3],"data": [[0,0,0.00077292737843196553],[0,1,0.0011138776045079284],[0,2,0.0011207335710646969],[1,0,0.064892294250092408],[1,1,0.065784300877997637],[1,2,0.071557140431312616],[2,0,0.83432469671001741],[2,1,0.7884942995675529],[2,2,0.81120733571064685],[3,0,0.10001008166145779],[3,1,0.14460752194994103],[3,2,0.11611479028697569]],"rows": [{"id": "No blast hit;Other", "metadata": null},{"id": "Root;Eurotiomycetes", "metadata": null},{"id": "Root;Saccharomycetes", "metadata": null},{"id": "Root;Sordariomycetes", "metadata": null}],"columns": [{"id": "Mock.2", "metadata": null},{"id": "Mock.3", "metadata": null},{"id": "Mock.1", "metadata": null}]}