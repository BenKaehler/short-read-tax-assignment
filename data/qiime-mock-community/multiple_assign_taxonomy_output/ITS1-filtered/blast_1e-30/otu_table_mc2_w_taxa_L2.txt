Taxon	Mock.2	Mock.3	Mock.1
No blast hit;Other	0.00762845716974	0.00563491023457	0.00774325012736
Root;Eurotiomycetes	0.0648922942501	0.065784300878	0.0715571404313
Root;Saccharomycetes	0.827469166919	0.783973266937	0.804584819154
Root;Sordariomycetes	0.100010081661	0.14460752195	0.116114790287
