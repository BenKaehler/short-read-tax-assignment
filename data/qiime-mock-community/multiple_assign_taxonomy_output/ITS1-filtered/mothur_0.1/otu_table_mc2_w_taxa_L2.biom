{"id": "None","format": "Biological Observation Matrix 1.0.0","format_url": "http://biom-format.org","type": "Taxon table","generated_by": "QIIME 1.6.0-dev","date": "2013-06-14T09:18:14.415122","matrix_type": "sparse","matrix_element_type": "float","shape": [10, 3],"data": [[0,0,6.7211076385388307e-05],[0,2,6.7923246731193751e-05],[1,1,6.5522212029878125e-05],[1,2,3.3961623365596876e-05],[2,0,0.06495950532647779],[2,1,0.065849823090027515],[2,2,0.071625063678043804],[3,1,6.5522212029878125e-05],[3,2,3.3961623365596876e-05],[4,0,0.00043687199650502403],[4,1,0.00072074433232865943],[4,2,0.00054338597384955001],[5,0,3.3605538192694154e-05],[5,1,6.5522212029878125e-05],[5,2,3.3961623365596876e-05],[6,0,3.3605538192694154e-05],[6,2,3.3961623365596876e-05],[7,0,0.83432469671001741],[7,1,0.78842877735552308],[7,2,0.8111733740872813],[8,0,0.10007729273784317],[8,1,0.14460752194994103],[8,2,0.11614875191034128],[9,0,6.7211076385388307e-05],[9,1,0.00019656663608963438],[9,2,0.00030565461029037188]],"rows": [{"id": "Root;Agaricomycetes", "metadata": null},{"id": "Root;Dothideomycetes", "metadata": null},{"id": "Root;Eurotiomycetes", "metadata": null},{"id": "Root;Incertae_sedis", "metadata": null},{"id": "Root;Other", "metadata": null},{"id": "Root;Pezizomycetes", "metadata": null},{"id": "Root;Pucciniomycetes", "metadata": null},{"id": "Root;Saccharomycetes", "metadata": null},{"id": "Root;Sordariomycetes", "metadata": null},{"id": "Root;Tremellomycetes", "metadata": null}],"columns": [{"id": "Mock.2", "metadata": null},{"id": "Mock.3", "metadata": null},{"id": "Mock.1", "metadata": null}]}