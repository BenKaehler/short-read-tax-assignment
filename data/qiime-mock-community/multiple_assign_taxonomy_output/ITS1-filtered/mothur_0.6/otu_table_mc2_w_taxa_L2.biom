{"id": "None","format": "Biological Observation Matrix 1.0.0","format_url": "http://biom-format.org","type": "Taxon table","generated_by": "QIIME 1.6.0-dev","date": "2013-06-14T10:03:02.593017","matrix_type": "sparse","matrix_element_type": "float","shape": [4, 3],"data": [[0,0,0.064892294250092408],[0,1,0.065784300877997637],[0,2,0.071557140431312616],[1,0,0.00090734953120274217],[1,1,0.0013104442405975629],[1,2,0.0013924265579894719],[2,0,0.83419027455724659],[2,1,0.78829773293146344],[2,2,0.81093564272372209],[3,0,0.10001008166145779],[3,1,0.14460752194994103],[3,2,0.11611479028697569]],"rows": [{"id": "Root;Eurotiomycetes", "metadata": null},{"id": "Root;Other", "metadata": null},{"id": "Root;Saccharomycetes", "metadata": null},{"id": "Root;Sordariomycetes", "metadata": null}],"columns": [{"id": "Mock.2", "metadata": null},{"id": "Mock.3", "metadata": null},{"id": "Mock.1", "metadata": null}]}