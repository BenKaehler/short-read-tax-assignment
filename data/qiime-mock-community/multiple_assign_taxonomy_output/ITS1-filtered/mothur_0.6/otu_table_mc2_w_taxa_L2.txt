Taxon	Mock.2	Mock.3	Mock.1
Root;Eurotiomycetes	0.0648922942501	0.065784300878	0.0715571404313
Root;Other	0.000907349531203	0.0013104442406	0.00139242655799
Root;Saccharomycetes	0.834190274557	0.788297732931	0.810935642724
Root;Sordariomycetes	0.100010081661	0.14460752195	0.116114790287
