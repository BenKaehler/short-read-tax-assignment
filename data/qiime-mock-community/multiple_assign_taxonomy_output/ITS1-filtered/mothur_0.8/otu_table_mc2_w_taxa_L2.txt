Taxon	Mock.2	Mock.3	Mock.1
Root;Eurotiomycetes	0.0648922942501	0.065784300878	0.0715571404313
Root;Other	0.0391168464563	0.0370200497969	0.0383087111564
Root;Saccharomycetes	0.795980777632	0.752588127375	0.774019358125
Root;Sordariomycetes	0.100010081661	0.14460752195	0.116114790287
