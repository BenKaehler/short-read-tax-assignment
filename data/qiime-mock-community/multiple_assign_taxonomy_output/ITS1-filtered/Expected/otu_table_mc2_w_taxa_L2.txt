Taxon	Mock.1	Mock.2	Mock.3
Root;Saccharomycetes	0.0625	0.0625	0.0625	
Root;Saccharomycetes	0.0625	0.0625	0.0625	
Root;Saccharomycetes	0.0625	0.0625	0.0625	
Root;Saccharomycetes	0.0625	0.0625	0.0625	
Root;Saccharomycetes	0.0625	0.0625	0.0625	
Root;Saccharomycetes	0.0625	0.0625	0.0625	
Root;Saccharomycetes	0.0625	0.0625	0.0625	
Root;Saccharomycetes	0.0625	0.0625	0.0625	
Root;Eurotiomycetes	0.0625	0.0625	0.0625	
Root;Eurotiomycetes	0.0625	0.0625	0.0625	
Root;Sordariomycetes	0.0625	0.0625	0.0625	
Root;Sordariomycetes	0.0625	0.0625	0.0625	
